articles/active-directory/develop/msal-net-xamarin-android-considerations.md

```diff
@@ -107,6 +106,102 @@ Here's an example of a class that represents the values of the XML file:
   }
 ```
 
+### System Browser Support in Brokered Authentication
+If you are using the system browser for interactive authentication, it is possible you will have configured your application to use brokered authentication when the device does not have broker installed. In this scenario, MSAL will try to authenticate using the default system browser in the device. This will fail out of the box because the redirect URI is configured for broker and the system browser would know how to use it to navigate back to MSAL. To resolve this, you can configure what is known as an intent filter with the broker redirect URI that you used in step four. You will need to modify your application's manifest to add the intent filter as shown below.
```

Comment:
> If you are using the system browser for interactive authentication, it is possible you will have configured your application to use brokered authentication when the device does not have broker installed.

Reword this whole section to be more *intent-based*. Again: goal first, _then_ the what/why/how. First address or note what the developer trying to do, then tell them how to accomplish that goal.

> configure what is known as an intent filter with the broker redirect URI that you used in step four

Which "step four" is this referring to?